web/webpack.config.dev.js

```diff
@@ -19,14 +20,14 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.(js|jsx)$/,
+        test: /\.[jt]sx?$/,
```

Comment:
intentional?  `jt`?

Reply:
Right now there is no typescript, but I was trying to standardize these so it's more versatile.  It would match js/ts/tsx/jsx